extensions/amp-story/1.0/amp-story.js

```diff
@@ -344,6 +346,21 @@ export class AmpStory extends AMP.BaseElement {
         createPseudoLocale(LocalizedStringsEn, (s) => `[${s} one two]`)
       );
 
+    // If there are inline strings, register as current document language.
+    const inlineStrings = this.win.document.querySelector(
+      'script[amp-strings="amp-story"]'
+    );
+    if (
+      inlineStrings &&
+      inlineStrings.getAttribute('i-amphtml-version') ==
+        getMode(this.win).rtvVersion
+    ) {
+      getLocalizationService(this.element).registerLocalizedStringBundle(
+        this.win.document.querySelector('[lang]')?.getAttribute('lang') || 'en',
+        JSON.parse(inlineStrings.textContent)
```

Comment:
This can fail and throw an error

Reply:
Using early returns now to tryParseJson (and don't register anything if it fails)